src/pytorch_lightning/tuner/lr_finder.py

```diff
@@ -16,8 +16,9 @@
 import os
 import uuid
 from functools import wraps
-from typing import Any, Dict, Optional, Sequence
+from typing import Any, Callable, Dict, List, Optional, Sequence, Union
 
+import matplotlib.pyplot as plt
```

Comment:
`matplotlib` is an optional dependency, and thus we cannot assume that it's always available, so we need to protect the import statement like this:

https://github.com/Lightning-AI/lightning/blob/cf189bdc9b32326ea02b9af6dd6e1bf04f0e4d2e/src/pytorch_lightning/strategies/deepspeed.py#L56-L58

Reply:
Thanks! I tried to replicate the same code for Matplotlib in 7645add.

Reply:
Whoops, I forgot to remove the actual import. It should be fixed now in 4d6068cd1300e4a991d36352d8111f175d9568ac.